doc/source/api/sequence_annotations.rst

```diff
@@ -0,0 +1,46 @@
+.. _sequence_annotations:
+
+************************
+Sequence Annotations API
+************************
+For the Sequence Annotation schema definitions, see `Sequence Annotation schema <../schemas/sequenceAnnotations.html>`_
+
+
+------------------------
+Feature Based Hierarchy
+------------------------
+The central object of the GA4GH Sequence Annotation API is a Feature.  The Feature describes an interval of interest on some reference(s).  It has a span from a start position to a stop position as well as descriptive data.  A Feature has one parent Feature, and can have an ordered array of child Features, which enables the construction of more complex representations in a hierarchical way.
+
+For example, a single gene Feature may be parent to several different transcript Features.  The specific exons for each transcript would have that transcript Feature as parent.  The same physical exon may occur as part of two different transcript Features, but in our notation, it would be
+encoded as two separate exon Features, each with a different parent, both occupying the same genomic coordinates. This structure can also exend to annotating CDS, binding sites or any other sub-gene level features.
+
+
+------------------------------
+The Sequence Annotation Schema
+------------------------------
+
+This model is similar to that used by the standard `GFF3`_ file format.
+
+.. _GFF3: http://sequenceontology.org/resources/gff3.html
+
+The main differences concern the deprecation and replacement of discontinuous features, the replacing
+of multi-parent features with multiple copies of that feature, and the ability to impose an explicit order on child features.
```

Comment:
Is that necessarily possible/relevant? For example in GFF a fusion transcript would have two parent genes. Duplicating the transcript seems inappropriate?